firmware/application/apps/ui_playlist.cpp

```diff
@@ -89,9 +89,9 @@ void PlaylistView::txtline_process(std::string &line) {
     uint32_t item_delay;
 
     current = line.find(',', previous);
-    if (current == std::string::npos){ //compatibility with old PPL grammar
+    if (current == std::string::npos) {  // compatibility with old PPL grammar
```

Comment:
Since find returns an 'npos' on fail, I think it the opposite that you want:

if (current == std::string::npos) {  // compatibility with old PPL grammar
        item_delay = 0;
    } else {
        item_delay = strtoll(line.substr(previous).c_str(), nullptr, 10);
    }

Reply:
It isn't . I was wrong ^^